tests/unit/commands/local/start_lambda/core/test_options.py

```diff
@@ -0,0 +1,11 @@
+from unittest import TestCase
+
+from samcli.commands.local.start_lambda.cli import cli
+from samcli.commands.local.start_lambda.core.options import ALL_OPTIONS
+
+
+class TestOptions(TestCase):
+    def test_all_options_formatted(self):
+        command_options = [param.human_readable_name for param in cli.params]
+        command_options = [command_option for command_option in command_options if command_option is not None]
+        self.assertEqual(sorted(ALL_OPTIONS), sorted(command_options))
```

Comment:
I'm a bit not sure if this is the right thing to assert equal. `cli.params` does not include options that's automatically added by click (e.g. `--help`). `cli.pararms` also includes arguments, which I believe would not be added to `ALL_OPTIONS`

Reply:
Open to a better testing strategy overall.

Reply:
I think we can just simply compare these two:
1. set of `ALL_OPTIONS` excluding "help"
2. set of `cli.params` excluding `click.core.Argument` type